Resources/doc/reference/configuration/invalidation.rst

```diff
@@ -20,7 +19,25 @@ routes when some other routes are requested.
                         villain_details:     # e.g., /villain/{id}
                             ignore_extra_params: false    # Defaults to true
 
-.. include:: /includes/enabled.rst
+.. include:: ../../includes/enabled.rst
```

Comment:
i think we had absolute paths on purpose. they are absolute in regard to the documentation root, so sphinx should be able to work with them. did you run into an issue?